docs/content/config/security/rspamd.md

```diff
@@ -92,6 +92,17 @@ Rspamd provides a [web interface][rspamd-docs::web-interface], which contains st
 
 ![Rspamd Web Interface](https://rspamd.com/img/webui.png)
 
+To make use of the web interface you need to setup setup a password otherwise you won't be able to log in.
+To do so you must
+[set the `password` option for the controller worker to a password hash generated by `rspamadm pw`](#with-the-help-of-a-custom-file).
+
+??? example "Configuration example"
+    Below is an example of how the configuration should look like.
+
+    ```bash
+    set-option-for-controller password "yourHashedPassword"
+    ```
+
```

Comment:
```suggestion
To use the web interface you will need to configure a password, otherwise you won't be able to log in.

??? example "Set a custom password"

    Add this line to [your rspamd `custom-commands.conf` config](#with-the-help-of-a-custom-file) which sets the `password` option of the _controller worker_:

    ```
    set-option-for-controller password "your hashed password here"
    ```

    The password hash can be generated via the `rspamadm pw` command:

    ```bash
    docker exec -it dms rspamadm pw
    ```

```